docs/data/data-grid/editing-legacy/editing.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Data Grid - Editing (legacy)
+---
+
+# Data Grid - Editing (legacy)
+
+<p class="description">The data grid has built-in edit capabilities that you can customize to your needs.</p>
+
+## Cell editing
+
+To enable cell editing within a column, set the `editable` property in the `GridColDef` object to `true` to allow editing cells of this column.
+
+```tsx
+<DataGrid columns={[{ field: 'name', editable: true }]} />
+```
+
+{{"demo": "BasicEditingGrid.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Start editing
+
+If a cell is editable and has focus, any of the following interactions will start the edit mode:
+
+- A <kbd class="key">Enter</kbd> keydown
+- A <kbd class="key">Backspace</kbd> or <kbd class="key">Delete</kbd> keydown. It will also delete the value and stops the edit mode instantly.
+- A keydown of any printable key, for instance `a`, `E`, `0`, or `$`
+- A double click on the cell
+- A call to `apiRef.current.setCellMode(id, field, 'edit')`.
+
+  ```tsx
+  /**
+    * Set the cellMode of a cell.
+    * @param GridRowId
+    * @param string
+    * @param 'edit' | 'view'
+    */
+  setCellMode: (id: GridRowId, field: string, mode: GridCellMode) => void;
+  ```
+
+### Stop editing
+
+If a cell is in edit mode and has focus, any of the following interactions will stop the edit mode:
+
+- A <kbd class="key">Escape</kbd> keydown. It will also roll back changes done in the value of the cell.
+- A <kbd class="key">Tab</kbd> keydown. It will also save and goes to the next cell on the same row.
+- A <kbd class="key">Enter</kbd> keydown. It will also save and goes to the next cell on the same column.
+- A mousedown outside the cell
+- A call to `apiRef.current.setCellMode(id, field, 'view')`.
+
+### Control cell editability
+
+In addition to the `editable` flag on columns, you can enable or disable editing of individual cells using the `isCellEditable` prop.
+
+In this demo, only the rows with an even `Age` value are editable.
+The editable cells have a green background for better visibility.
+
+{{"demo": "IsCellEditableGrid.js", "bg": "inline"}}
+
+### Controlled editing
+
+The `editRowsModel` prop lets you control the editing state.
+You can use the `onEditRowsModelChange` callback to control the `GridEditRowsModel` state.
```

Comment:
Add colon to indicate that this paragraph offers information about the code sample that follows.

```suggestion
You can use the `onEditRowsModelChange` callback to control the `GridEditRowsModel` state:
```